### PR TITLE
Diego/delete controller

### PR DESCRIPTION
**Pull request by lDiego257**
The Delete controller was implemented using TDD. It also includes its implementation into the api.yml.

### CODE REVIEW

File: backend/server/api/services/todos.service.js [hunk 1]
```diff
@@ -18,5 +18,19 @@ export class ToDoService {
     const newTodo = allTodos.child(newTodoRef.key).toJSON();
     return { ...newTodo, key: newTodoRef.key };
   }
+  async deleteByID(key) {
+    var ref = this.db.ref("todo/" + key);
+    var success = await ref.once("value").then(function (snapshot) {
```
Comment (fernandogruning) [inline backend/server/api/services/todos.service.js:23]: You can omit the `.then()` and just directly put its body. Also by doing that, you wouldn't need to use a `success` variable

File: backend/server/api/services/todos.service.js [hunk 2]
```diff
@@ -18,5 +18,19 @@ export class ToDoService {
     const newTodo = allTodos.child(newTodoRef.key).toJSON();
     return { ...newTodo, key: newTodoRef.key };
   }
+  async deleteByID(key) {
+    var ref = this.db.ref("todo/" + key);
```
Comment (fernandogruning) [inline backend/server/api/services/todos.service.js:22]: Set as `const`

File: backend/server/api/controllers/todo/controller.js [hunk 3]
```diff
@@ -22,6 +22,19 @@ export class Controller {
       }
     }
   }
+
+  async deleteByID(req, res) {
+    try {
+      var a = await ToDoService.deleteByID(req.params.id);
```
Comment (fernandogruning) [inline backend/server/api/controllers/todo/controller.js:28]: Set as `const`. Maybe use a more descriptive var name

**Review by fernandogruning — APPROVED** [review]
LGTM, do the proposed changes and then merge

**Review by kevinrosarioc — APPROVED** [review]

**Pull request merged**